recipes/mmg/meta.yaml

```diff
@@ -28,11 +28,11 @@ requirements:
     - jom  # [win]  
     - libscotch
     - iscd-commons
-    - {{ pin_subpackage("iscd-linearelasticity", exact=True) }}    
+    - iscd-linearelasticity  
   run:
     - libscotch
     - iscd-commons
-    - {{ pin_subpackage("iscd-linearelasticity", exact=True) }}
+    - iscd-linearelasticity
```

Comment:
You cannot use `pin_subpackage` here because "iscd-linearelasticity" is not a package that is defined in this recipe. i.e. it is not a subpackage in this file.

You would use `pin_compatible` in `requirements/run` if you want an externally defined dependency to be compatible in the run and host sections.